cmd/containerboot/serve.go

```diff
@@ -68,7 +68,6 @@ func watchServeConfigChanges(ctx context.Context, path string, cdChanged <-chan
 		if prevServeConfig != nil && reflect.DeepEqual(sc, prevServeConfig) {
 			continue
 		}
-		validateHTTPSServe(certDomain, sc)
```

Comment:
Removed this function call as it's called from within `updateServeConfig` on the next line anyway